EpiResponsivePicture/TypeScript/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "epi-responsive-picture",
```

Comment:
since this is more Optimizely then epi now, maybe we should call it optimizely-responsive-picture?

Reply:
I will change npm packaga name and leave nuget as epi for the historical reasons